src/simulation/elements/HETR.cpp

```diff
@@ -0,0 +1,77 @@
+#include "simulation/Elements.h"
+//#TPT-Directive ElementClass Element_HETR PT_HETR 666
+Element_HETR::Element_HETR()
+{
+	Identifier = "DEFAULT_PT_HETR";
+	Name = "HETR";
```

Comment:
General comment on the name, I don't like it. It will make people think it heats things, but it's actually only FILT related.